starters/next-wordpress-starter/next.config.js

```diff
@@ -38,6 +38,17 @@ if (process.env.WPGRAPHQL_URL === undefined) {
 // remove trailing slash if it exists
 imageDomain = imageDomain.replace(/\/$/, '');
 
+let PANTHEON_ENVIRONMENT, IS_LIVE_ENVIRONMENT;
+if (process.env.PANTHEON_ENVIRONMENT_URL) {
+	const envPrefix = process.env.PANTHEON_ENVIRONMENT_URL.match(/^([^-]*)/)[0];
+	PANTHEON_ENVIRONMENT =
+		process.env.PANTHEON_ENVIRONMENT_URL.match(/^([^-]*-)[^-]*/)[0];
```

Comment:
Not sure this is matching an environment like `multi-demo-test`

Reply:
That is correct it would not match that right now. How can we differentiate between the ending of a multidev name and the beginning of the site name? Since everything is divided by `-` I'm not sure how we can tell the difference dynamically.

Reply:
Good question. Not sure there is a good way? Might be a limitation we need to document.
@backlineint any suggestions?
Also while I'm pinging you in, should the prefix contain the `multi-` prefix or should we lop that off? I'm thinking this isn't much good for dev and test if you need to name your FE site branch `multi-test` and then this doesn't work.

Reply:
So the case where the backend has a different site name, would we expect the user to lop off their FE site name from `PANTHEON_ENVIRONMENT`?

Reply:
@backlineint @CobyPear Just tried to implement this idea to take everything before `.appa.pantheon.site` and am running into an issue with testing. It seems Pantheon has a character limit on multidev names so taking the entire prefix + site name and using that in naming the multidev environment is not viable.
.
<img width="642" alt="Screen Shot 2022-12-06 at 2 18 58 PM" src="https://user-images.githubusercontent.com/39682014/206002296-6e4809f7-11ac-4d18-99fd-df2b2eda2d9f.png">

Any thoughts on how to proceed?

Reply:
Hmm yeah 11 chars is pretty short. This is making me wonder if we can ask the BE to expose this env var for us given the complexity of parsing it out. I don't remember if there was something preventing that on their side or if we just need to ask again.

Reply:
I agree, assuming there is nothing preventing that on their side I think that is the smoothest path forward. Should I move this ticket to blocked and add a note?

Reply:
That makes sense to me